### DISCUSSION TITLE
IPED Linux tutorials

### DISCUSSION THREAD

**Discussion by niviosouza** [sepinf-inc/IPED#2074] [Show and tell]
Guys, I'm creating detailed tutorials in Portuguese regarding IPED in Linux environments. The content is at https://github.com/niviosouza/IPED. There is already a ready-made tutorial on building IPED on Linux (contributions are always welcome). The use of the IPED docker container on Linux has not yet started and all help is welcome. Feel free to contribute to both tutorials and others if you wish. Thanks!

**Comment by lfcnassif** [comment]
Thanks @niviosouza. Have you considered to write your tutorials in English? I think it could reach a larger number of Linux users. If yes, we can create a section in our wiki and give write permission to you.

**Comment by niviosouza** [comment]
Hello Luis. Yes I can write tutorials in English.

Em ter., 6 de fev. de 2024 às 16:09, Luis Filipe Nassif <
***@***.***> escreveu:

> Thanks @niviosouza <https://github.com/niviosouza>. Have you considered
> to write your tutorials in English? I think it could reach a larger number
> of Linux users. If yes, we can create a section in our wiki and give write
> permission to you.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/sepinf-inc/IPED/discussions/2074#discussioncomment-8386868>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABF5HKVEMLKMXUQG22FRSZLYSJ5VVAVCNFSM6AAAAABC4NZHFKVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DGOBWHA3DQ>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

Atenciosamente,

-- 
____________________________________
Nivio Paula de Souza
+++++++++++++++++++++++
Cel: +55 12 99605-9700
Skypeme: capnivio
***@***.***
+++++++++++++++++++++++

echo
"Ck5pdmlvIFBhdWxhIGRlIFNvdXphCisrKysrKysrKysrKysrKysrKysrKysrCkNlbDogKzU1IDEyIDk5NjA1LTk3MDAKU2t5cGVtZTogY2Fwbml2aW8KY2Fwbml2aW9AZ21haWwuY29tCisrKysrKysrKysrKysrKysrKysrKysrCgo="
| base64 -d

**Comment by niviosouza** [comment]
Luis, se for possível, dê uma olhada no tutorial que já existe e veja se há
algo a corrigir ou melhorar. Depois disso, podemos traduzir o tutorial para
o inglês. O que acha?

Em ter., 6 de fev. de 2024 às 17:39, Nivio ***@***.***> escreveu:

> Hello Luis. Yes I can write tutorials in English.
>
> Em ter., 6 de fev. de 2024 às 16:09, Luis Filipe Nassif <
> ***@***.***> escreveu:
>
>> Thanks @niviosouza <https://github.com/niviosouza>. Have you considered
>> to write your tutorials in English? I think it could reach a larger number
>> of Linux users. If yes, we can create a section in our wiki and give write
>> permission to you.
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/sepinf-inc/IPED/discussions/2074#discussioncomment-8386868>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/ABF5HKVEMLKMXUQG22FRSZLYSJ5VVAVCNFSM6AAAAABC4NZHFKVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DGOBWHA3DQ>
>> .
>> You are receiving this because you were mentioned.Message ID:
>> ***@***.***>
>>
>
>
> --
>
> Atenciosamente,
>
> --
> ____________________________________
> Nivio Paula de Souza
> +++++++++++++++++++++++
> Cel: +55 12 99605-9700
> Skypeme: capnivio
> ***@***.***
> +++++++++++++++++++++++
>
> echo
> "Ck5pdmlvIFBhdWxhIGRlIFNvdXphCisrKysrKysrKysrKysrKysrKysrKysrCkNlbDogKzU1IDEyIDk5NjA1LTk3MDAKU2t5cGVtZTogY2Fwbml2aW8KY2Fwbml2aW9AZ21haWwuY29tCisrKysrKysrKysrKysrKysrKysrKysrCgo="
> | base64 -d
>


-- 

Atenciosamente,

-- 
____________________________________
Nivio Paula de Souza
+++++++++++++++++++++++
Cel: +55 12 99605-9700
Skypeme: capnivio
***@***.***
+++++++++++++++++++++++

echo
"Ck5pdmlvIFBhdWxhIGRlIFNvdXphCisrKysrKysrKysrKysrKysrKysrKysrCkNlbDogKzU1IDEyIDk5NjA1LTk3MDAKU2t5cGVtZTogY2Fwbml2aW8KY2Fwbml2aW9AZ21haWwuY29tCisrKysrKysrKysrKysrKysrKysrKysrCgo="
| base64 -d

**Reply by lfcnassif** [reply]
Seems good to me. But I'm not the right person for this Linux task. @arisjr or @aberenguel, do you have some available time to help reviewing @niviosouza tutorial?

**Reply by lfcnassif** [reply]
Or maybe @patrickdalla could have time to review?

**Comment by patrickdalla** [comment]
@arisjr already have a docker project to create a linux image with IPED pre requisites and IPED itself https://github.com/iped-docker/iped. Although it seems it does not have a human language descriptive installation step by step, whats is good for anyone who needs to understand and customize its own image/installation.

For example: I use an docker image customized, that is different from the one @arisjr had at the time I had created. So I used it as a reference to create my own image, and I suggested some changes in this his project. Its docker image had IPED embedded. My image have only IPED prerequisites installed, with the IPED itself being referenced from a mapped volume. It helped me to update IPED without update the docker Image. Also, my docker image runs like a service that monitors a mounted volume and processes any new image put there. It seems @arisjr accepted my suggestions and separated the docker project in subprojects: [Dockerfile.dependencies], [Dockerfile.processor] and others, what seems good but I haven't used yet.

Anyway, it seems many dependencies are missing in the tutorial in https://github.com/niviosouza/IPED, maybe because they already come with Linux Mint 21.3 which I don't know. I remember I had to compile many of them in my CentOS installation and in my UBUNTU docker image. Check https://github.com/iped-docker/iped , as it is complete. The tutorial mentions only https://github.com/libyal/libagdb.git. but the @arisjr is compiling https://github.com/libyal/libbfio, https://github.com/libyal/libvslvm, https://github.com/libyal/libvmdk, https://github.com/libyal/libvhdi, https://github.com/libyal/libewf-legacy, https://github.com/sshock/AFFLIBv3, and many others.

**Comment by niviosouza** [comment]
In the tutorial I created, only libraries that did not exist in LinuxMint were compiled (sleuthkit, jep (4.0.3), numpy and libagdb). The libraries https://github.com/libyal/libbfio, https://github.com/libyal/libvslvm, https://github.com/libyal/libvmdk, https://github.com/libyal/libvhdi, https://github.com/libyal/libewf-legacy, https://github.com/sshock/AFFLIBv3 mentioned by @patrickdalla were installed through LinuxMint packages and therefore were not compiled, as presented in the tutorial in command :

apt install ant automake automake1.11 autopoint bellsoft-java11-full build-essential ewf-tools git graphviz imagemagick libafflib-dev libesedb-utils libevt-utils libevtx1 libevtx-dev libevtx-utils libewf-dev libjavafxsvg-java libmsiecf-utils libopenjfx-java libopenjfx-java-doc libopenjfx-jni libparse-win32registry-perl libpff1 libreoffice-gtk2 libreoffice-java-common libscca-utils libssl-dev libtool libvhdi-dev libvmdk-dev libvslvm-dev m4 make maven mplayer mplayer-gui openjdk-11-dbg openjdk-11-jdk openjdk-11-jre openjdk-11-source openjfx pkg-config python3.10-venv python3-evtx python3-libevtx python3-libewf python3-pip python3-vlc rifiuti2 smbclient tesseract-ocr tesseract-ocr-eng tesser act -ocr-by tesseract-ocr-script-latn vim vim-addon-manager vim-runtime vim-scripts vlc

Please, if there is still a library that is missing, report it.

The libraries installed by default in the LinuxMint 21.3 installation are the following:

liba52-0.7.4 libaa1 libabsl20210324 libabw-0.1-1 libaccountsservice0 libacl1 libafflib0v5 libaio1 libaliased-perl libann0 libao-common libao4 libaom3 libaopalliance-java libapache-pom-java libapparmor1 libappstream-glib8 libappstream4 libapt-pkg-perl libapt-pkg6.0 libarchive-zip-perl libarchive13 libargon2-1 libaribb24-0 libasan6 libasan8 libasound2 libasound2-data libasound2-plugins libaspell15 libass9 libassuan0 libasyncns0 libatasmart4 libatinject-jsr330-api-java libatk-adaptor libatk-bridge2.0-0 libatk1.0-0 libatk1.0-data libatkmm-1.6-1v5 libatm1 libatomic1 libatopology2 libatspi2.0-0 libattr1 libaudio2 libaudit-common libaudit1 libauthen-sasl-perl libavahi-client3 libavahi-common-data libavahi-common3 libavahi-core7 libavahi-glib1 libavc1394-0 libavcodec-extra libavcodec-extra58 libavdevice58 libavfilter7 libavformat58 libavutil56 libayatana-appindicator3-1 libayatana-ido3-0.4-0 libayatana-indicator3-7 libb-hooks-endofscope-perl libb-hooks-op-check-perl libbabeltrace1 libbatik-java libbfio-dev libbfio1 libbinutils libblas3 libblkid1 libblockdev-crypto2 libblockdev-fs2 libblockdev-loop2 libblockdev-part-err2 libblockdev-part2 libblockdev-swap2 libblockdev-utils2 libblockdev2 libbluetooth3 libbluray2 libboost-filesystem1.74.0 libboost-iostreams1.74.0 libboost-locale1.74.0 libboost-regex1.74.0 libboost-thread1.74.0 libbpf0 libbrasero-media3-1 libbrlapi0.8 libbrotli1 libbs2b0 libbsd0 libburn4 libbz2-1.0 libc-bin libc-dev-bin libc-devtools libc6 libc6-dbg libc6-dev libcaca0 libcairo-gobject-perl libcairo-gobject2 libcairo-perl libcairo2 libcairomm-1.0-1v5 libcamel-1.2-63 libcanberra-gtk3-0 libcanberra-gtk3-module libcanberra-pulse libcanberra0 libcap-ng0 libcap2 libcap2-bin libcapture-tiny-perl libcaribou-common libcaribou0 libcbor0.8 libcc1-0 libcddb2 libcdi-api-java libcdio-cdda2 libcdio-paranoia2 libcdio19 libcdparanoia0 libcdr-0.1-1 libcdt5 libcgi-fast-perl libcgi-pm-perl libcgraph6 libchamplain-0.12-0 libchamplain-gtk-0.12-0 libchromaprint1 libcinnamon-control-center1 libcinnamon-desktop-dbg libcinnamon-desktop4 libcinnamon-menu-3-0 libcinnamon-menu-3-0-dbg libcjs-dbg libcjs0 libclass-data-inheritable-perl libclass-method-modifiers-perl libclass-xsaccessor-perl libclone-perl libclucene-contribs1v5 libclucene-core1v5 libclutter-1.0-0 libclutter-1.0-common libclutter-gst-3.0-0 libclutter-gtk-1.0-0 libcodec2-1.0 libcogl-common libcogl-pango20 libcogl-path20 libcogl20 libcolamd2 libcolord2 libcolorhug2 libcom-err2 libcommons-cli-java libcommons-io-java libcommons-lang3-java libcommons-logging-java libcommons-parent-java libconfig-tiny-perl libconst-fast-perl libcpanel-json-xs-perl libcrack2 libcrypt-dev libcrypt1 libcryptsetup12 libcscreensaver0 libctf-nobfd0 libctf0 libcups2 libcupsfilters1 libcupsimage2 libcurl3-gnutls libcurl4 libcvc-dbg libcvc0 libcwidget4 libdaemon0 libdata-dpath-perl libdata-dump-perl libdata-messagepack-perl libdata-optlist-perl libdata-validate-domain-perl libdata-validate-ip-perl libdata-validate-uri-perl libdatrie1 libdav1d5 libdazzle-1.0-0 libdazzle-common libdb5.3 libdbus-1-3 libdbus-glib-1-2 libdbusmenu-glib4 libdbusmenu-gtk3-4 libdc1394-25 libdca0 libdconf1 libde265-0 libdebconfclient0 libdebuginfod-common libdebuginfod1 libdecor-0-0 libdee-1.0-4 libdeflate0 libdevel-callchecker-perl libdevel-size-perl libdevel-stacktrace-perl libdevmapper-event1.02.1 libdevmapper1.02.1 libdjvulibre-text libdjvulibre21 libdmapsharing-3.0-2 libdmraid1.0.0.rc16 libdns-export1110 libdotconf0 libdouble-conversion3 libdpkg-perl libdrm-amdgpu1 libdrm-common libdrm-intel1 libdrm-nouveau2 libdrm-radeon1 libdrm2 libdv4 libdvbpsi10 libdvdcss2 libdvdnav4 libdvdread8 libdw1 libdynaloader-functions-perl libe-book-0.1-1 libebackend-1.2-10 libebml5 libebook-1.2-20 libebook-contacts-1.2-3 libecal-2.0-1 libecryptfs1 libedata-book-1.2-26 libedata-cal-2.0-1 libedataserver-1.2-26 libedataserverui-1.2-3 libedit2 libefiboot1 libefivar1 libegl-mesa0 libegl1 libel-api-java libelf1 libemail-address-xs-perl libenca0 libenchant-2-2 libencode-locale-perl libeot0 libepoxy0 libept1.6.0 libepubgen-0.1-1 liberror-perl libesedb1 libespeak-ng1 libestr0 libetonyek-0.1-1 libevdev2 libevent-2.1-7 libevt1 libewf2 libexception-class-perl libexempi8 libexif12 libexiv2-27 libexpat1 libexpat1-dev libexporter-tiny-perl libext2fs2 libexttextcat-2.0-0 libexttextcat-data libextutils-depends-perl libfaad2 libfakeroot libfastjson4 libfcgi-perl libfcgi0ldbl libfdisk1 libffi8 libffmpegthumbnailer4v5 libfftw3-double3 libfftw3-single3 libfido2-1 libfile-basedir-perl libfile-desktopentry-perl libfile-fcntllock-perl libfile-find-rule-perl libfile-listing-perl libfile-mimeinfo-perl libflac8 libflatpak0 libflite1 libfluidsynth3 libfont-afm-perl libfont-ttf-perl libfontconfig1 libfontembed1 libfontenc1 libfprint-2-2 libfreeaptx0 libfreehand-0.1-1 libfreetype6 libfribidi0 libfuse2 libfuse3-3 libfwupd2 libfwupdplugin5 libgail-3-0 libgail-common libgail18 libgamemode0 libgamemodeauto0 libgbm1 libgc1 libgcab-1.0-0 libgcc-11-dev libgcc-12-dev libgcc-s1 libgck-1-0 libgcr-base-3-1 libgcr-ui-3-1 libgcrypt20 libgd3 libgdata-common libgdata22 libgdbm-compat4 libgdbm6 libgdk-pixbuf-2.0-0 libgdk-pixbuf2.0-bin libgdk-pixbuf2.0-common libgee-0.8-2 libgeoclue-2-0 libgeocode-glib0 libgeronimo-annotation-1.3-spec-java libgeronimo-interceptor-3.0-spec-java libgexiv2-2 libgfortran5 libgif7 libgirepository-1.0-1 libgl1 libgl1-mesa-dri libglapi-mesa libgles2 libglib-object-introspection-perl libglib-perl libglib2.0-0 libglib2.0-bin libglib2.0-data libglibmm-2.4-1v5 libglu1-mesa libglvnd0 libglx-mesa0 libglx0 libgme0 libgmp10 libgnome-desktop-3-19 libgnomekbd-common libgnomekbd8 libgnutls30 libgoa-1.0-0b libgoa-1.0-common libgoa-backend-1.0-1 libgomp1 libgpg-error0 libgpgme11 libgpgmepp6 libgphoto2-6 libgphoto2-l10n libgphoto2-port12 libgpm2 libgpod-common libgpod4 libgraphene-1.0-0 libgraphite2-3 libgrilo-0.3-0 libgs9 libgs9-common libgsf-1-114 libgsf-1-common libgsf-bin libgsm1 libgsound0 libgspell-1-2 libgspell-1-common libgssapi-krb5-2 libgssdp-1.2-0 libgstreamer-gl1.0-0 libgstreamer-plugins-bad1.0-0 libgstreamer-plugins-base1.0-0 libgstreamer-plugins-good1.0-0 libgstreamer1.0-0 libgtk-3-0 libgtk-3-bin libgtk-3-common libgtk-layer-shell0 libgtk2.0-0 libgtk2.0-bin libgtk2.0-common libgtk3-perl libgtkmm-3.0-1v5 libgtksourceview-3.0-1 libgtksourceview-3.0-common libgtksourceview-4-0 libgtksourceview-4-common libgtop-2.0-11 libgtop2-common libgts-0.7-5 libgts-bin libguava-java libgucharmap-2-90-7 libgudev-1.0-0 libguice-java libgupnp-1.2-1 libgupnp-igd-1.0-4 libgusb2 libgutenprint-common libgutenprint9 libgvc6 libgvpr2 libgweather-3-16 libgweather-common libgxps2 libhandy-1-0 libharfbuzz-icu0 libharfbuzz0b libhawtjni-runtime-java libheif1 libhogweed6 libhpmud0 libhsqldb1.8.0-java libhtml-form-perl libhtml-format-perl libhtml-html5-entities-perl libhtml-parser-perl libhtml-tagset-perl libhtml-tree-perl libhttp-cookies-perl libhttp-daemon-perl libhttp-date-perl libhttp-message-perl libhttp-negotiate-perl libhunspell-1.7-0 libhyphen0 libibus-1.0-5 libical3 libice-dev libice6 libicu70 libidn12 libidn2-0 libiec61883-0 libieee1284-3 libigdgmm12 libijs-0.35 libilmbase25 libimagequant0 libimobiledevice-utils libimobiledevice6 libimport-into-perl libinih1 libinput-bin libinput10 libinstpatch-1.0-2 libio-html-perl libio-interactive-perl libio-prompt-tiny-perl libio-socket-ssl-perl libio-string-perl libip4tc2 libip6tc2 libipc-run3-perl libipc-system-simple-perl libipt2 libisc-export1105 libisl23 libisofs6 libiterator-perl libiterator-util-perl libitm1 libiw30 libixml10 libjack-jackd2-0 libjansi-java libjansi-native-java libjansson4 libjavascriptcoregtk-4.0-18 libjavascriptcoregtk-4.1-0 libjaxp1.3-java libjbig0 libjbig2dec0 libjcat1 libjpeg-turbo8 libjpeg8 libjs-jquery libjs-jquery-hotkeys libjs-jquery-isonscreen libjs-jquery-metadata libjs-jquery-tablesorter libjs-jquery-throttle-debounce libjs-mathjax libjs-sphinxdoc libjs-underscore libjson-c5 libjson-glib-1.0-0 libjson-glib-1.0-common libjson-maybexs-perl libjsp-api-java libjsr305-java libjte2 libjxr-tools libjxr0 libk5crypto3 libkate1 libkeybinder-3.0-0 libkeyutils1 libklibc libkmod2 libkpathsea6 libkrb5-3 libkrb5support0 libksba8 liblab-gamut1 liblangtag-common liblangtag1 liblapack3 liblcms2-2 liblcms2-utils libldacbt-enc2 libldap-2.5-0 libldb2 liblept5 liblibreoffice-java liblightdm-gobject-1-0 liblilv-0-0 liblirc-client0 liblist-compare-perl liblist-moreutils-perl liblist-moreutils-xs-perl liblist-someutils-perl liblist-utilsby-perl libllvm15 liblmdb0 liblocale-gettext-perl liblouis-data liblouis20 liblouisutdml-bin liblouisutdml-data liblouisutdml9 liblqr-1-0 liblsan0 libltc11 libltdl-dev libltdl7 liblua5.2-0 liblvm2cmd2.03 liblwp-mediatypes-perl liblwp-protocol-https-perl liblz4-1 liblzma5 liblzo2-2 libmad0 libmagic-mgc libmagic1 libmagickcore-6.q16-6 libmagickcore-6.q16-6-extra libmagickwand-6.q16-6 libmalcontent-0-0 libmanette-0.2-0 libmarkdown2 libmate-desktop-2-17 libmate-menu2 libmate-panel-applet-4-1 libmateweather-common libmateweather1 libmatroska7 libmaven-parent-java libmaven-resolver-java libmaven-shared-utils-java libmaven3-core-java libmaxminddb0 libmbim-glib4 libmbim-proxy libmd0 libmd4c0 libmessaging-menu0 libmetacity3 libmfx1 libmhash2 libminiupnpc17 libmjpegutils-2.1-0 libmm-glib0 libmng2 libmnl0 libmodplug1 libmodule-implementation-perl libmodule-runtime-perl libmoo-perl libmoox-aliases-perl libmount1 libmouse-perl libmozjs-102-0 libmp3lame0 libmpc3 libmpcdec6 libmpdec3 libmpeg2-4 libmpeg2encpp-2.1-0 libmpfr6 libmpg123-0 libmplex2-2.1-0 libmpv1 libmsiecf1 libmspack0 libmspub-0.1-1 libmtdev1 libmtp-common libmtp-runtime libmtp9 libmuffin0 libmujs1 libmusicbrainz5-2 libmusicbrainz5cc2v5 libmwaw-0.3-3 libmysofa1 libmythes-1.2-0 libnamespace-clean-perl libnatpmp1 libnautilus-extension1a libncurses6 libncursesw6 libndp0 libnemo-extension1 libneon27-gnutls libnet-dbus-perl libnet-domain-tld-perl libnet-http-perl libnet-ipv6addr-perl libnet-libidn-perl libnet-netmask-perl libnet-ssleay-perl libnetaddr-ip-perl libnetfilter-conntrack3 libnetpbm10 libnetplan0 libnettle8 libnewt0.52 libnfnetlink0 libnfs13 libnftables1 libnftnl11 libnghttp2-14 libnice10 libnl-3-200 libnl-genl-3-200 libnl-route-3-200 libnm0 libnma-common libnma0 libnorm1 libnotify-bin libnotify4 libnpth0 libnsl-dev libnsl2 libnspr4 libnss-mdns libnss-myhostname libnss-systemd libnss3 libntfs-3g89 libnuma1 libnumber-compare-perl libnvpair3linux libodfgen-0.1-1 libogg0 libopenal-data libopenal1 libopencore-amrnb0 libopencore-amrwb0 libopenexr25 libopengl0 libopenh264-6 libopenjp2-7 libopenmpt-modplug1 libopenmpt0 libopenni2-0 libopus0 liborc-0.4-0 liborcus-0.17-0 liborcus-parser-0.17-0 libostree-1-1 libp11-kit0 libpackage-stash-perl libpackagekit-glib2-18 libpagemaker-0.0-0 libpam-cap libpam-gnome-keyring libpam-modules libpam-modules-bin libpam-runtime libpam-systemd libpam0g libpango-1.0-0 libpangocairo-1.0-0 libpangoft2-1.0-0 libpangomm-1.4-1v5 libpangoxft-1.0-0 libpaper-utils libpaper1 libparams-classify-perl libparams-util-perl libparted-fs-resize0 libparted2 libpath-tiny-perl libpathplan4 libpcap0.8 libpcaudio0 libpci3 libpciaccess0 libpcre2-16-0 libpcre2-32-0 libpcre2-8-0 libpcre3 libpcsclite1 libpeas-1.0-0 libpeas-common libperl5.34 libperlio-gzip-perl libperlio-utf8-strict-perl libpgm-5.3-0 libphonenumber8 libpipeline1 libpipewire-0.3-0 libpipewire-0.3-modules libpixman-1-0 libpkcs11-helper1 libplacebo192 libplexus-cipher-java libplexus-classworlds-java libplexus-component-annotations-java libplexus-interpolation-java libplexus-sec-dispatcher-java libplexus-utils2-java libplist3 libplymouth5 libpng16-16 libpocketsphinx3 libpolkit-agent-1-0 libpolkit-gobject-1-0 libpoppler-cpp0v5 libpoppler-glib8 libpoppler118 libpopt0 libpostproc55 libproc-processtable-perl libprocps8 libprotobuf-lite23 libprotobuf23 libproxy-tools libproxy1-plugin-gsettings libproxy1-plugin-networkmanager libproxy1v5 libpsl5 libpthread-stubs0-dev libpugixml1v5 libpulse-mainloop-glib0 libpulse0 libpulsedsp libpwquality-common libpwquality1 libpython3-dev libpython3-stdlib libpython3.10 libpython3.10-dev libpython3.10-minimal libpython3.10-stdlib libqmi-glib5 libqmi-proxy libqpdf28 libqrencode4 libqt5core5a libqt5dbus5 libqt5gui5 libqt5network5 libqt5svg5 libqt5widgets5 libqt5x11extras5 libquadmath0 librabbitmq4 libraptor2-0 libraqm0 librasqal3 libraw1394-11 libraw20 librda-common librda0 librdf0 libreadline8 libreoffice-base libreoffice-base-core libreoffice-base-drivers libreoffice-calc libreoffice-common libreoffice-core libreoffice-draw libreoffice-gnome libreoffice-gtk3 libreoffice-help-common libreoffice-help-de libreoffice-help-en-gb libreoffice-help-en-us libreoffice-help-es libreoffice-help-fr libreoffice-help-it libreoffice-help-pt libreoffice-help-pt-br libreoffice-help-ru libreoffice-help-zh-cn libreoffice-help-zh-tw libreoffice-impress libreoffice-l10n-de libreoffice-l10n-en-gb libreoffice-l10n-en-za libreoffice-l10n-es libreoffice-l10n-fr libreoffice-l10n-it libreoffice-l10n-pt libreoffice-l10n-pt-br libreoffice-l10n-ru libreoffice-l10n-zh-cn libreoffice-l10n-zh-tw libreoffice-math libreoffice-sdbc-hsqldb libreoffice-style-colibre libreoffice-writer libresid-builder0c2a librest-0.7-0 librevenge-0.0-0 librhythmbox-core10 librole-tiny-perl librsvg2-2 librsvg2-common librtmp1 librubberband2 libruby3.0 libsamplerate0 libsane-common libsane-hpaio libsane1 libsasl2-2 libsasl2-modules libsasl2-modules-db libsasl2-modules-gssapi-mit libsbc1 libscca1 libsdl-image1.2 libsdl1.2debian libsdl2-2.0-0 libseccomp2 libsecret-1-0 libsecret-common libselinux1 libsemanage-common libsemanage2 libsensors-config libsensors5 libsepol2 libserd-0-0 libsereal-decoder-perl libsereal-encoder-perl libservlet-api-java libservlet3.1-java libsgutils2-2 libshine3 libshout3 libsidplay1v5 libsidplay2 libsigc++-2.0-0v5 libsigsegv2 libsisu-inject-java libsisu-plexus-java libsixel1 libslang2 libslf4j-java libsm-dev libsm6 libsmartcols1 libsmbclient libsmbios-c2 libsnapd-glib1 libsnappy1v5 libsndfile1 libsndio7.0 libsnmp-base libsnmp40 libsodium23 libsonic0 libsord-0-0 libsort-versions-perl libsoundtouch1 libsoup-3.0-0 libsoup-3.0-common libsoup-gnome2.4-1 libsoup2.4-1 libsoup2.4-common libsource-highlight-common libsource-highlight4v5 libsoxr0 libspa-0.2-modules libspandsp2 libspatialaudio0 libspectre1 libspeechd2 libspeex1 libspeexdsp1 libsphinxbase3 libsqlite3-0 libsratom-0-0 libsrt1.4-gnutls libsrtp2-1 libss2 libssh-4 libssh-gcrypt-4 libssh2-1 libssl3 libstartup-notification0 libstdc++-11-dev libstdc++6 libstemmer0d libstrictures-perl libsub-exporter-perl libsub-exporter-progressive-perl libsub-identify-perl libsub-install-perl libsub-name-perl libsub-quote-perl libsuitesparseconfig5 libswresample3 libswscale5 libsyntax-keyword-try-perl libsystemd0 libtag1v5 libtag1v5-vanilla libtalloc2 libtasn1-6 libtcl8.6 libtdb1 libteamdctl0 libterm-readkey-perl libtesseract4 libtevent0 libtext-charwidth-perl libtext-glob-perl libtext-iconv-perl libtext-levenshteinxs-perl libtext-markdown-discount-perl libtext-wrapi18n-perl libtext-xslate-perl libthai-data libthai0 libtheora0 libtie-ixhash-perl libtiff5 libtime-duration-perl libtime-moment-perl libtimedate-perl libtimezonemap-data libtimezonemap1 libtinfo6 libtirpc-common libtirpc-dev libtirpc3 libtotem-plparser-common libtotem-plparser18 libtry-tiny-perl libtsan0 libtsan2 libtspi1 libtss2-esys-3.0.2-0 libtss2-mu0 libtss2-sys1 libtss2-tcti-cmd0 libtss2-tcti-device0 libtss2-tcti-mssim0 libtss2-tcti-swtpm0 libtwolame0 libubsan1 libuchardet0 libudev1 libudfread0 libudisks2-0 libunicode-utf8-perl libunistring2 libunity-protocol-private0 libunity-scopes-json-def-desktop libunity9 libuno-cppu3 libuno-cppuhelpergcc3-3 libuno-purpenvhelpergcc3-3 libuno-sal3 libuno-salhelpergcc3-3 libunoloader-java libunshield0 libunwind8 libupnp13 libupower-glib3 liburi-perl liburing2 libusb-1.0-0 libusbmuxd6 libuuid-perl libuuid1 libuutil3linux libuv1 libv4l-0 libv4lconvert0 libva-drm2 libva-wayland2 libva-x11-2 libva2 libvariable-magic-perl libvdpau1 libvhdi1 libvidstab1.1 libvisio-0.1-1 libvisual-0.4-0 libvisual-0.4-plugins libvlc-bin libvlc5 libvlccore9 libvmdk1 libvncclient1 libvo-aacenc0 libvo-amrwbenc0 libvolume-key1 libvorbis0a libvorbisenc2 libvorbisfile3 libvorbisidec1 libvpx7 libvslvm1 libvte-2.91-0 libvte-2.91-common libvulkan1 libwacom-bin libwacom-common libwacom9 libwagon-file-java libwagon-http-shaded-java libwagon-provider-api-java libwavpack1 libwayland-client0 libwayland-cursor0 libwayland-egl1 libwayland-server0 libwbclient0 libwebkit2gtk-4.0-37 libwebkit2gtk-4.1-0 libwebp7 libwebpdemux2 libwebpmux3 libwebrtc-audio-processing1 libwebsocket-api-java libwildmidi2 libwmf-0.2-7 libwmf-0.2-7-gtk libwmf0.2-7-gtk libwmflite-0.2-7 libwnck-3-0 libwnck-3-common libwoff1 libwpd-0.10-10 libwpe-1.0-1 libwpebackend-fdo-1.0-1 libwpg-0.3-3 libwps-0.4-4 libwrap0 libwww-perl libwww-robotrules-perl libx11-6 libx11-data libx11-dev libx11-protocol-perl libx11-xcb1 libx264-163 libx265-199 libxalan2-java libxapian30 libxapp1 libxatracker2 libxau-dev libxau6 libxaw7 libxcb-composite0 libxcb-dri2-0 libxcb-dri3-0 libxcb-glx0 libxcb-icccm4 libxcb-image0 libxcb-keysyms1 libxcb-present0 libxcb-randr0 libxcb-render-util0 libxcb-render0 libxcb-res0 libxcb-shape0 libxcb-shm0 libxcb-sync1 libxcb-util1 libxcb-xfixes0 libxcb-xinerama0 libxcb-xinput0 libxcb-xkb1 libxcb-xv0 libxcb1 libxcb1-dev libxcomposite1 libxcursor1 libxcvt0 libxdamage1 libxdmcp-dev libxdmcp6 libxdo3 libxerces2-java libxext6 libxfixes3 libxfont2 libxft2 libxi6 libxinerama1 libxkbcommon-x11-0 libxkbcommon0 libxkbfile1 libxkbregistry0 libxklavier16 libxml-commons-external-java libxml-commons-resolver1.1-java libxml-libxml-perl libxml-namespacesupport-perl libxml-parser-perl libxml-sax-base-perl libxml-sax-expat-perl libxml-sax-perl libxml-twig-perl libxml-xpathengine-perl libxml2 libxmlb2 libxmlgraphics-commons-java libxmlsec1 libxmlsec1-nss libxmlsec1-openssl libxmu6 libxmuu1 libxpm4 libxpresent1 libxrandr2 libxreaderdocument3 libxreaderview3 libxrender1 libxres1 libxs-parse-keyword-perl libxshmfence1 libxslt1.1 libxss1 libxt-dev libxt6 libxtables12 libxtst6 libxv1 libxvidcore4 libxvmc1 libxxf86dga1 libxxf86vm1 libxxhash0 libyajl2 libyaml-0-2 libyaml-libyaml-perl libyaml-tiny-perl libyelp0 libzbar0 libzfs4linux libzimg2 libzip4 libzmq5 libzpool5linux libzstd1 libzvbi-common libzvbi0 libzxingcore1

**Comment by patrickdalla** [comment]
Right, it's good to know that Mint is such a 'IPED friendly' linux distro for the desktop user, and for the tutorial scope it is enough. 

Although, for the docker version, to create a small and clean image, starting from the smallest existent Linux image is desirable.
